docs/releasenotes/index.rst

```diff
@@ -10,6 +10,14 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
+v1.3
+====
+
+* Flocker API has new endpoints to create, list, renew, and delete lease on a dataset.
+* Flocker now works on servers where ``SELinux`` is enabled.
+* Flocker is more accurate in figuring out attached block device path on ``Cinder`` backend.
+* Flocker now has smaller log footprint from ``EBS`` storage driver on success path.
```

Comment:
Instead I copied the sentence we used when we fixed a similar bug in the EBS driver.
And I removed the EBS reduced logging line.